components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodProjectManager.kt

```diff
@@ -65,17 +79,23 @@ class GitpodProjectManager(
                 val projectRootManager = ProjectRootManager.getInstance(project)
                 if (projectRootManager.projectSdk == null) {
                     projectRootManager.projectSdk = sdk
-                    thisLogger().warn("gitpod: '${project.name}' project: SDK was auto preconfigured: $sdk")
+                    thisLogger().warn("gitpod: '${project.name}' project: " +
+                            "SDK was auto preconfigured: ${projectRootManager.projectSdk}")
+                } else {
+                    thisLogger().warn("gitpod: '${project.name}' project: " +
+                            "SDK was already configured: ${projectRootManager.projectSdk}")
                 }
             }
         }
         for (module in ModuleManager.getInstance(project).modules) {
             ModuleRootModificationUtil.updateModel(module) { m ->
                 if (m.sdk == null) {
                     m.sdk = sdk
-                    thisLogger().warn("gitpod: '${module.name}' module: SDK was auto preconfigured: $sdk")
+                    thisLogger().warn("gitpod: '${module.name}' module: SDK was auto preconfigured: ${m.sdk}")
+                } else {
+                    thisLogger().warn("gitpod: '${module.name}' module: SDK was already configured: ${m.sdk}")
```

Comment:
let remove it, it is unnecessary noise, or change to debug level that it it not visible in default output

Reply:
Removed!